### PR TITLE
Add Commits and Branches links to More dropdown

### PR DESCRIPTION
**Pull request by snowe2010**
Add Commits, Branches, Contributors, and License links to More dropdown

Add 3 new icons to icons.tsx for the dropdown

test url:
https://github.com/sindresorhus/refined-github

From conversation [here](https://github.com/sindresorhus/refined-github/pull/1822#issuecomment-469425666)

### CODE REVIEW

File: source/features/more-dropdown.tsx [hunk 1]
```diff
@@ -45,6 +45,26 @@ async function init() {
...
+		</a>,
+
+		<a href={`/${repoUrl}/graphs/contributors`} class="rgh-reponav-more dropdown-item" data-skip-pjax>
+			{icons.organization()}
+			{' Contributors'}
+		</a>,
+
+		<a href={`/${repoUrl}/blob/master/license`} class="rgh-reponav-more dropdown-item" data-skip-pjax>
+			{icons.law()}
+			{' License'}
```
Comment (bfred-it) [inline source/features/more-dropdown.tsx]: Contributors and license aren't useful enough to appear on all repo pages.

File: source/libs/icons.tsx [hunk 2]
```diff
@@ -40,6 +42,10 @@ export const branch = (): SVGElement => <svg aria-hidden="true" class="octicon o
 
 export const foldDown = (): SVGElement => <svg aria-hidden="true" class="octicon octicon-fold-down" width="14" height="16" ><path fill-rule="evenodd" d="M4 11l3 3 3-3H8V5H6v6H4zm-4 0c0 .55.45 1 1 1h2.5l-1-1h-1l2-2H5V8H3.5l-2-2H5V5H1c-.55 0-1 .45-1 1l2.5 2.5L0 11zm10.5-2H9V8h1.5l2-2H9V5h4c.55 0 1 .45 1 1l-2.5 2.5L14 11c0 .55-.45 1-1 1h-2.5l1-1h1l-2-2z"/></svg>;
 
+export const organization = (): SVGElement => <svg className="octicon octicon-organization" viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M16 12.999c0 .439-.45 1-1 1H7.995c-.539 0-.994-.447-.995-.999H1c-.54 0-1-.561-1-1 0-2.634 3-4 3-4s.229-.409 0-1c-.841-.621-1.058-.59-1-3 .058-2.419 1.367-3 2.5-3s2.442.58 2.5 3c.058 2.41-.159 2.379-1 3-.229.59 0 1 0 1s1.549.711 2.42 2.088C9.196 9.369 10 8.999 10 8.999s.229-.409 0-1c-.841-.62-1.058-.59-1-3 .058-2.419 1.367-3 2.5-3s2.437.581 2.495 3c.059 2.41-.158 2.38-1 3-.229.59 0 1 0 1s3.005 1.366 3.005 4z"></path></svg>;
```
Comment (FloEdelmann) [inline source/libs/icons.tsx]: I think those new icons are not needed anymore, are they?

Comment (snowe2010) [reply source/libs/icons.tsx]: Do you want me to remove them or leave them for later use? I'm sure others will use them at some point.

Comment (bfred-it) [reply source/libs/icons.tsx]: Remove for now, there’s no way to tell if they’ll ever be used

File: source/features/more-dropdown.tsx [hunk 3]
```diff
@@ -32,20 +31,25 @@ async function init() {
 
 	select('.reponav-dropdown .dropdown-menu').append(
 		<a href={`/${repoUrl}/compare`} class="rgh-reponav-more dropdown-item" data-skip-pjax>
-			{icons.darkCompare()}
-			{' Compare'}
+			{icons.darkCompare()} Compare
 		</a>,
 
 		isEnterprise() ? '' :
```
Comment (bfred-it) [inline source/features/more-dropdown.tsx]: @busches is this still necessary?

**Comment by bfred-it** [comment]
@sindresorhus last check for the readme wording

**Comment by snowe2010** [comment]
Oh I didn't see that 'needs readme update' label, or I would have done that. I'm sorry.

**Comment by bfred-it** [comment]
Thanks for the PR, @snowe2010 💯

File: source/features/more-dropdown.tsx (hunk 3 above)
Comment (busches) [reply source/features/more-dropdown.tsx]: We upgraded this weekend, let me check tomorrow!

Comment (busches) [reply source/features/more-dropdown.tsx]: Still don't have dependencies link in `2.16.4` :/

**Pull request merged**